app/controllers/agent/EnterClientsUTRController.scala

```diff
@@ -45,7 +46,8 @@ class EnterClientsUTRController @Inject()(enterClientsUTR: EnterClientsUTR,
                                           clientDetailsService: ClientDetailsService,
                                           val authorisedFunctions: FrontendAuthorisedFunctions,
                                           val authActions: AuthActions,
-                                          val auditingService: AuditingService)
+                                          val auditingService: AuditingService,
+                                          val ITSAStatusService: ITSAStatusService)
```

Comment:
if we can revert change to original as we are not using ITSAStatusService in this controller